### PR TITLE
valid token may not have subject field

### PR DESCRIPTION
**Pull request by binjip978**

### CODE REVIEW

File: logging/interceptor.go [hunk 1]
```diff
@@ -218,6 +218,11 @@ func addAccountIDField(ctx context.Context, fields logrus.Fields) error {
 func addCustomField(ctx context.Context, fields logrus.Fields, customField string) error {
 	field, err := auth.GetJWTField(ctx, customField, nil)
 	if err != nil {
+		if customField == DefaultSubjectKey {
+			// non client token does not have subject ingore
```
Comment (addudko) [inline logging/interceptor.go:222]:
```suggestion
			// non client token does not have subject field, ignore
```

**Review by addudko — APPROVED** [review]

File: logging/interceptor.go [hunk 2]
```diff
@@ -218,6 +218,11 @@ func addAccountIDField(ctx context.Context, fields logrus.Fields) error {
 func addCustomField(ctx context.Context, fields logrus.Fields, customField string) error {
 	field, err := auth.GetJWTField(ctx, customField, nil)
 	if err != nil {
+		if customField == DefaultSubjectKey {
+			// non client token does not have subject field, ignore
+			return nil
+		}
+
```
Comment (drewwells) [inline logging/interceptor.go:225]: Updates to this project need PR tests. Please add a unit test verifying what this functionality does

**Review by drewwells — CHANGES_REQUESTED** [review]

**Review by Evgeniy-L — APPROVED** [review]
Looks good

**Review by drewwells — APPROVED** [review]
Thanks for unit test! sorry on slow review

**Pull request merged**